gm4_mob_curing/data/gm4_mob_curing/functions/init.mcfunction

```diff
@@ -0,0 +1,11 @@
+scoreboard objectives add gm4_mob_cu_data dummy
```

Comment:
```suggestion
scoreboard objectives add gm4_mob_curing_data dummy
```
As of Minecraft 1.18 there is no longer a length limit for objective names. Note that this change will have to be made in multiple places if you accept it.